### DISCUSSION TITLE
Where is doc website source code?

### DISCUSSION THREAD

**Discussion by sospartan** [pocketbase/pocketbase#36] [Q&A]
Found some typo there, hope I can contribute to it .

**Comment by ganigeorgiev** [comment]
The documentation is not auto generated and it is part of the static landing site (built with SvelteKit).

I didn't bother open source it because it is kinda messy and a little complicated to be edited by users not familiar with the codebase, but I'll try to find some time in the future and may publish it.

In the meantime, if you find typos or think that the examples could be improved, feel free to open an issue or discussion I'll fix them.

**Reply by sospartan** [reply]
Got it. Not big problem for now, I'll wait for the source code.
You can search 'dabase' in the source code to fix it. :)

**Reply by jimafisk** [reply]
I noticed a typo on https://pocketbase.io/docs/manage-collections/. I believe "pointint" should be "pointing"

**Comment by bukowa** [comment]
Is there any way to search docs?

**Reply by ganigeorgiev** [reply]
@bukowa Unfortunately at the moment the site documentation doesn't have a search index. I have it in my todo but that will be probably after the support for the JS app hook bindings because the docs will have to be restructured anyway.

_If you haven't already, you can find the site documentation repo in https://github.com/pocketbase/site._
